src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -683,6 +683,75 @@ TEST_F(HMICapabilitiesTest, SetIviCooperating) {
   EXPECT_EQ(true, hmi_capabilities_test->is_ivi_cooperating());
 }
 
+TEST_F(
+    HMICapabilitiesTest,
+    UpdateCapabilitiesDependingOn_ccpuVersion_FromCacheForOld_RequestForNew) {
+  MockApplicationManager mock_app_mngr;
+  event_engine_test::MockEventDispatcher mock_dispatcher;
+  MockApplicationManagerSettings mock_application_manager_settings;
+  std::string ccpu_version = "4.1.3.B_EB355B";
+  std::string ccpu_version_new = "5.1.3.B_EB355B";
+  const std::string hmi_capabilities_not_valid_file =
+      "hmi_capabilities_not_valid.json";
```

Comment:
@ydementieiev file can be renamed as well

Reply:
@AByzhynar [done](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/ccc7535da6372fd01e1cbb9171d0c170af9087c2#diff-c1d0a5558de3ab3e31ce8813484ebefeR694)